kie-server-parent/kie-server-tests/kie-server-integ-tests-dmn-pmml/pom.xml

```diff
@@ -0,0 +1,303 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.kie.server</groupId>
+    <artifactId>kie-server-tests</artifactId>
+    <version>7.45.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>kie-server-integ-tests-dmn-pmml</artifactId>
+
+  <name>KIE :: Execution Server :: Tests :: DMN-PMML Integration Tests</name>
+  <description>KIE Execution Server Integration Tests (REST, JMS) with configuration options to run on different containers.</description>
+
+  <properties>
+    <java.module.name>org.kie.server.itests.dmn</java.module.name>
```

Comment:
maybe for completeness (even if not actually used) you might want to revise the JPMS module name here, isn't this conflicting with an existing one?

Reply:
@tarilabs 
Thank you for finding this. It should be fixed now.